packages/c/claws-mail/monitoring.yml

```diff
@@ -0,0 +1,9 @@
+releases:
+  id: 293
+  rss: https://github.com/Tux/claws-mail/tags.atom
+security:
+  cpe:
+    - vendor: claws-mail
```

Comment:
Are these duplicated for a reason?

Reply:
Wasn't sure if claws-mail would be affected by both components or not.